build.sbt

```diff
@@ -99,6 +99,9 @@ lazy val RootProject = Project(id = "root", base = file("."))
 
 lazy val ColossusProject: Project = Project(id = "colossus", base = file("colossus"))
   .settings(ColossusSettings: _*)
+  .settings{
+    mimaPreviousArtifacts := Set("com.tumblr" % "colossus_2.11" % "0.9.1")
```

Comment:
maybe put the version in a constant so we only have to update it in one place next time.